python/ray/worker.py

```diff
@@ -2205,6 +2208,17 @@ def method(self):
         retry_exceptions (bool): Only for *remote functions*. This specifies
             whether application-level errors should be retried
             up to max_retries times.
+        scheduling_strategy (SchedulingStrategy): Strategy about how to
+            schedule a remote function or actor. Possible values are
+            None: ray will figure out the scheduling strategy to use, it
+            will either be the PlacementGroupSchedulingStrategy using parent's
+            placement group if parent has one and has
+            placement_group_capture_child_tasks set to true,
+            or the DefaultSchedulingStrategy;
+            "DEFAULT" or `DefaultSchedulingStrategy`:
```

Comment:
I don't think we should recommend two ways to specify DEFAULT, how about just the string?